quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/hostname/DefaultHostnameProvider.java

```diff
@@ -354,4 +357,9 @@ private <T> T fromBaseUriOrDefault(Function<URI, T> resolver, URI baseUri, T def
 
         return defaultValue;
     }
+
+    @Override
+    public boolean isSupported() {
```

Comment:
Should we rather implement the new non-deprecated `isSupported(Config.Scope config)`?

Reply:
This was due to me waffling between the first two options listed in https://github.com/keycloak/keycloak/pull/26003#discussion_r1450346314 - let's make sure we agree on the approach and then I'll fix this method to match.

Reply:
> Based upon the above it sounds like we'll leave the default and have the method still be deprecated, so I'll change this to the other signature.

Makes sense to me.